docs/content/reference/install-configuration/tls/spiffe.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Traefik SPIFFE Documentation"
+description: "Learn how to configure Traefik to use SPIFFE. Read the technical documentation."
+---
+
+# SPIFFE
+
+Secure the backend connection with SPIFFE.
+{: .subtitle }
+
+[SPIFFE](https://spiffe.io/docs/latest/spiffe-about/overview/) (Secure Production Identity Framework For Everyone), 
+provides a secure identity in the form of a specially crafted X.509 certificate, 
+to every workload in an environment.
+
+Traefik is able to connect to the Workload API to obtain an x509-SVID used to secure the connection with SPIFFE enabled backends.
+
+## Configuration Example
+
+Enabling SPIFFE can be done by using a file (YAML or TOML) or CLI arguments.
+
+```yaml tab="File (YAML)"
+# Default Servers Transport
+serversTransport:
+  spiffe:
+    ids:
+    - spiffe://trust-domain/id1
+    - spiffe://trust-domain/id2
+    trustDomain: "spiffe://trust-domain"
+spiffe:
+    workloadAPIAddr: localhost
+```
+
+```yaml tab="ServerTransport"
+# Override the default Servers Transport on the Services that reference it.
+apiVersion: traefik.io/v1alpha1
+kind: ServersTransport
+metadata:
+  name: mytransport
+  namespace: apps
+spec:
+  spiffe:
+    ids:
+    - spiffe://trust-domain/specific-id1
+    trustDomain: "spiffe://trust-domain"
+```
```

Comment:
The server transport resource is a dynamic configuration so I am not sure this should be documented here, @nmengin @sheddy-traefik wdyt ?

```suggestion
```